packages/core/src/texture/Texture2DArray.ts

```diff
@@ -26,14 +26,16 @@ export class Texture2DArray extends Texture {
    * @param length - Texture length
    * @param format - Texture format. default `TextureFormat.R8G8B8A8`
    * @param mipmap - Whether to use multi-level texture
+   * @param isSRGBColorSpace - Whether to use sRGB color space, default is true
```

Comment:
```ts
@param isSRGBColorSpace - Whether the texture data is in sRGB color space or linear color space. @defaultValue `true`
```

Reply:
Whether the texture data is in sRGB color space. @defaultValue `true`